src/tools/periph/bootnorm.S

```diff
@@ -20,9 +20,9 @@ bootnormal_code:
 real_start:     cli
```

Comment:
But with this CLI why do we care?
Is this just a cosmetic change?

Reply:
In debuggers, the IF=0 doesn't inhibit trace interrupts. To correctly operate there, the mov-to-ss locking should be used.